CODEOWNERS

```diff
@@ -162,6 +162,7 @@ homeassistant/components/pvoutput/* @fabaff
 homeassistant/components/qnap/* @colinodell
 homeassistant/components/quantum_gateway/* @cisasteelersfan
 homeassistant/components/qwikswitch/* @kellerza
+homeassistant/components/raincloud/* @tchellomello @vanstinator
```

Comment:
Please do not add other user as code owner if their name was not there before.

Reply:
Ok. I'll change it. My only thought was they're still a maintainer. But I can see how adding other users could be a problem :)

Reply:
done